src/index.js

```diff
@@ -1 +1,18 @@
 import './index.css';
+import Game from './modules/new_game.js';
+
+const form = document.getElementById('form');
+const game = new Game();
+const refreshButton = document.getElementById('refresh');
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const user = form.user.value;
+  const score = form.score.value;
+  game.addScore(user, score);
+  game.fetchScores();
```

Comment:
- [OPTIONAL] I would encourage you to empty the input fields at the next line to this. You can take this for example:
```
form.user.value = ""
form.score.value = "" 
```
The reason is for user of your app to know that the submission has been made and the input fields would be empty.